static-assets/components/cstudio-common/common-api.js

```diff
@@ -4493,6 +4501,26 @@ var nodeOpen = false,
                 YConnect.asyncRequest('GET', this.createServiceUri(serviceUrl), serviceCallback);
             },
 
+            /**
+             * get Quick Create
+             */
+            getQuickCreate: function(callback) {
+                var serviceUrl = this.getQuickCreateURL;
+                serviceUrl += "?siteId=" + CStudioAuthoringContext.site;
+
+                var serviceCallback = {
```

Comment:
Use new react app `ajax` lib. Like so:
```js
CrafterCMSNext.Util.ajax.get(this.createServiceUri(serviceUrl))
  .subscribe(
    function (response) {
      var results = response.response;
      results = results.items;
      callback.success(results);
    },
    function (response) {
      callback.failure(response);
      return null;
    }
  );
```

Reply:
PR is now merged: https://github.com/craftercms/studio-ui/pull/1193

Reply:
Done